### PR TITLE
Re-design proteome + Update examples for the Search box + Implement content for the landing page + Remove WIP message

### PR DESCRIPTION
**Pull request by p3rcypj**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/24qjegb, https://app.clickup.com/t/24x9dzj, https://app.clickup.com/t/24qjj7b, https://app.clickup.com/t/24xdwc7

### :memo: Implementation

Changed proteome SVG: restyled, inverted order, removed synonyms, removed ORF9c, moved NSP11 into ORF1a. When some protein/gen is clicked, the proteome hides. The previous fake margins were also "fixed". Search examples have been also updated. Implemented content for the landing page. Removed warning message from all pages.

### :art: Screenshots

image
image
image
image
image

### CODE REVIEW

File: app/assets/javascripts/covid19/src/webapp/components/structures-table/SearchBar.tsx [hunk 1]
```diff
@@ -37,20 +38,16 @@ export const SearchBar: React.FC<SearchBarProps> = React.memo(props => {
     } = props;
     const [open, setOpen] = React.useState(false);
     const [stateValue, setValueDebounced] = useDebouncedSetter(value, setValue, { delay: 500 });
-    const [autoSuggestionOptions, setAutoSuggestionOptions] = React.useState<Array<string>>([
-        "6YOR",
-        "Homo sapiens",
-        "SARS-CoV-2",
-    ]);
+    const [autoSuggestionOptions, setAutoSuggestionOptions] = React.useState<Array<string>>(
```
Comment (tokland) [inline app/assets/javascripts/covid19/src/webapp/components/structures-table/SearchBar.tsx:41]: [veeery minor, no need to change] 1) `string[]` is more idiomatic. But more complex `Array<{x: number, y: string}>` are ok. 2) When the value is inferred, we don't usually force the type in useState.

File: app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx [hunk 2]
```diff
@@ -15,6 +15,16 @@ import { HtmlTooltip } from "./HtmlTooltip";
...
 
+export const searchExamples = [
+    "EMD-21375",
+    "Glycoprotein",
+    "Remdesivir",
+    "3CL-Pro",
+    "Mouse",
+    "PanDDA",
+    "Martinez",
+];
```
Comment (tokland) [inline app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx:26]: Some words are lowercase in the task (ie. "mouse"), I guess it's not that important.

File: app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx [hunk 3]
```diff
@@ -15,6 +15,16 @@ import { HtmlTooltip } from "./HtmlTooltip";
 import i18n from "../../../utils/i18n";
 import "./Toolbar.css";
 
+export const searchExamples = [
```
Comment (tokland) [inline app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx:18]: Nice, good to abstract these arrays.

**Review by tokland — APPROVED** [review]
Looks good to me!

File: app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx (hunk 3 above)
Comment (p3rcypj) [reply app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx:18]: Thanks! 😊

File: app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx (hunk 2 above)
Comment (p3rcypj) [reply app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx:26]: Search is case insensitive, so I turned search examples uppercase for better looking. Should I follow more explicitly what has been asked? @adrianq

Comment (adrianq) [reply app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx:26]: Yes, please, in this case, in particular, I think it is important to follow the client requirements as they have been requested.

**Pull request merged**